src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java

```diff
@@ -0,0 +1,78 @@
+package org.folio.circulation.domain.validation;
+
+import java.util.Objects;
+import java.util.function.Predicate;
+
+import org.folio.circulation.domain.Request;
+import org.folio.circulation.domain.reorder.ReorderRequest;
+import org.folio.circulation.resources.context.ReorderRequestContext;
+import org.folio.circulation.support.BadRequestFailure;
+import org.folio.circulation.support.RecordNotFoundFailure;
+import org.folio.circulation.support.Result;
+import org.folio.circulation.support.request.RequestHelper;
+
+public class RequestQueueValidation {
+
+  private RequestQueueValidation() {
+    throw new UnsupportedOperationException("Do not instantiate");
+  }
+
+  public static Result<ReorderRequestContext> queueFoundForItem(
```

Comment:
How can a queue not exist for an item? Under what circumstances can that happen?